modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step12_rgmb_hetero.md

```diff
@@ -0,0 +1,90 @@
+       id=tutorial04-lfr_pinmesh
+       caption=Example LFR assembly pin cell mesh colored by `subdomain_id`.
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec12_rgmb_het_lfr/rgmb_lfr_assembly.i
+         id=tutorial04-rgmb_lfr-pin1
+         caption=LFR RGMB pin example.
+         block=Mesh/pin1
+
+`region_ids` is a 2-dimensional array where each row assigns to radial layer `region_ids` on a given axial layer, with the bottom layer defined first. Each pin has multiple radial regions (hole, fuel, clad, coolant) which must be defined for each axial layer.
+
+## AssemblyMeshGenerator
+
+[AssemblyMeshGenerator.md] takes the 7 pin types previously defined and places them into a regular hexagonal grid. Additionally, coolant and duct regions need to be added around the pins in order to create the assembly geometry.
+
+The assembly pitch has already been defined in [ReactorMeshParams.md] and forms the outer edge of the assembly duct. The inner half pitch of the duct should be specified using the `duct_halfpitch` parameter. Additional duct blocks can be created by adding more radii values to this parameter, remembering that each value must be less than the assembly `half_pitch`. The radial meshing subintervals for the duct are defined with the `duct_intervals` option.
+
+Whatever area remains between the outer pin boundaries and the inner half pitch of the duct is filled in with "background" region (representing coolant, in this case). The radial meshing subintervals for the coolant are defined with the `background_intervals` option.
+
+Extrusion is performed by using the `extrude`=`true` option and is performed after the 2D assembly geometry has been completed.
+
+!listing sec12_rgmb_het_lfr/rgmb_lfr_assembly.i
+         id=tutorial04-rgmb_lfr-assembly
+         caption=LFR RGMB assembly example.
+         block=Mesh/assembly
+
+RGMB allows assignment of background coolant and duct region IDs which are added when pins are patterned into an assembly lattice.
+
+!media tutorial04_meshing/rgmb_lfr_assembly.png
+       id=tutorial04-rgmb_lfr_assembly
+       caption=RGMB-generated LFR assembly mesh colored by `subdomain_id` (left), `pin_id` (middle) and `plane_id` (right).
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!media tutorial04_meshing/rgmb_lfr_assembly_matid.png
+       id=tutorial04-rgmb_lfr_assembly_matid
+       caption=`material_id` Reporting ID assignment.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+## Use of RGMB Mesh with Griffin
+
+`region_id` extra element integer needs to be renamed to `material_id` so that Griffin can recognize these values are material assignments.
+
+!listing sec12_rgmb_het_lfr/rgmb_lfr_assembly.i
+         id=tutorial04-rgmb_lfr-material-id
+         caption=LFR RGMB Material ID setup.
+         block=Mesh/lfr_assy
+
+Boundary conditions are assigned to the outer boundary sidesets.
```

Comment:
@shikhar413 - Please check my suggestion 

```suggestion
Boundary conditions are assigned to the outer boundary sidesets. The default outer boundary name for an assembly with type 1 is 'outer_assembly_1'.
```

Reply:
Suggested change:

Boundary conditions are assigned to the outer boundary sidesets. The default outer boundary name for an assembly with [!param](/Mesh/AssemblyMeshGenerator/assembly_type) 1 is `outer_assembly_1`.